specification/07-theory_operations.adoc

```diff
@@ -431,4 +431,98 @@ TSM ->> VMM: [COVH] - spdm_covh_connect_device()
 
 === Interface Assignment
 
+Once both the SPDM session and the IDE link are secured and established, the
+host supervisor domain manager may directly assign a TDI to a TVM, through the
+`COVH` interface. This is a four steps process:
+
+1. The host supervisor domain manager initiates the interface assignment flow by
+   having the TSM move the TDI into the TDISP `LOCKED` state.
+2. The TVM verifies and accepts the locked TDI into its TCB.
+3. The host supervisor domain manager asks the TSM to move the TDI to the TDISP
```

Comment:
why not let TVM move the TDI to RUN state directly?

Reply:
Of course it's the TVM. Fixed now.